lib/compress/index.js

```diff
@@ -464,19 +464,45 @@ def_optimize(AST_Node, function(self) {
     return self;
 });
 
-AST_Toplevel.DEFMETHOD("drop_console", function() {
+AST_Toplevel.DEFMETHOD("drop_console", function(options) {
+    var isBoolean = typeof options === "boolean";
+    var isArray = Array.isArray(options);
+
+    if (!isArray && !isBoolean) return;
```

Comment:
```suggestion
    var isArray = Array.isArray(options);
```

We don't need to validate options here, since TS definitions and the docs can help people pass in the correct option.